firebase-common/src/androidMain/AndroidManifest.xml

```diff
@@ -1 +1,4 @@
-<manifest package="dev.gitlive.firebase.common"/>
+<manifest package="dev.gitlive.firebase.common"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <application android:networkSecurityConfig="@xml/network_security_config"/>
+</manifest>
```

Comment:
can this config be added to test target only?

Reply:
Yes, I've added AndroidManifest for test target only.